### DISCUSSION TITLE
github environment tab in not visible in private repos

### DISCUSSION THREAD

**Discussion by konnecty-anand** [community/community#77285] [Actions]
### Select Topic Area

Question

### Body

I want to configure add & configure environment in github for manual approval and deployment in production for private repository.
I have attached the screenshot In the left sidebar "Environments" tab I don't see to configure it. 
Note: I'm owner of the organisation, so please someone help on this which I'm missing to change the setting
 
<img width="1157" alt="image" src="https://github.com/community/community/assets/149065413/b480694b-d38e-4526-bc01-93efa50aaba8">

**Comment by elderguardian** [comment]
Hey, this is a paid feature. It is described [here](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#about-environments):

```
Note: Users with GitHub Free plans can only configure environments for public repositories
If you convert a repository from public to private, any configured protection rules or 
environment secrets will be ignored, and you will not be able to configure any environments.
If you convert your repository back to public, you will have access to any previously 
configured protection rules and environment secrets.
```

**Comment by konnecty-anand** [comment]
thank you for providing information.

**Reply by olegshirko** [reply]
Doesn't work anymore
